layers/cmd/valueset.go

```diff
@@ -311,6 +313,50 @@ func LoadValuesetFiles(vs *Valuesets, files []string) error {
 	return nil
 }
 
+func (vsets Valuesets) LoadSpreadsheets(ctx *ls.Context) error {
+	for _, sheet := range vsets.Spreadsheets {
+		spreadsheet, err := cmdutil.ReadSpreadsheetFile(sheet)
+		if err != nil {
+			return fmt.Errorf("While reading file: %s, %w", spreadsheet, err)
+		}
+		sheetHeaders := make(map[string]string)
+		determineHeaderType := func(s string) string {
+			s = strings.TrimSpace(s)
+			suffix := s[len(s)-2:] // (i) or (o)
+			switch suffix {
+			case "(i)":
+				return "input"
+			case "(o)":
+				return "output"
+			}
+			return fmt.Sprint("Cannot determine header type")
```

Comment:
Ignore header if you cannot determine type. There are inputs, output, and ignored cols.